docker/run-docker-vircadia-web.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+# Start the metaverseserver with persistant data in local dir
+
+BASE=$(pwd)
+cd "${BASE}"
+
+DVERSION=latest
+
+docker run -it --rm \
```

Comment:
When I first ran the image, I got the error:
```
vircadia@nyxx:~/vircadia-web-mb/docker> ./run-docker-vircadia-web.sh
Log file: /home/cadia/vircadia-web/log/20220213.1741.log
/home/cadia/run-vircadia-web.sh: line 16: /home/cadia/vircadia-web/log/20220213.1741.log: Permission denied
vircadia@nyxx:~/vircadia-web-mb/docker>
```
which says that the mounted log directory didn't have the permissions to create files in the mounted directory.
This run file should check and/or change the permissions of the target log directory to allow the Docker account "cadia" to write into the mounted dir.

Reply:
ok. need to change how it runs the app. For now it requires sudo. I am going to drop 'sudo' and move app to /home/cadia. Let me do it.